airflow/providers/databricks/hooks/databricks_sql.py

```diff
@@ -242,9 +243,11 @@ def run(
 
     @staticmethod
     def _make_serializable(result):
-        """Transform the databricks Row objects into a JSON-serializable list of rows."""
-        if result is not None:
+        """Transform the databricks Row objects into JSON-serializable lists."""
+        if isinstance(result, list):
```

Comment:
statement = f"DESCRIBE TABLE {table.name};"
hook.run(statement, parameters=parameters, handler=lambda x: x.fetchall())

with this code `hook.run()` will return `[val1, val2, val3]` instead of `[row(), row(), row()]` isn't this breaking change??